README.md

```diff
@@ -79,11 +80,12 @@ Then you can fetch changes at any time from this remote:
 git pull upstream develop
 ```
 
-When you have finished editing, commit and push to your fork:
```

Comment:
Just check - do we always recommend forking? Users often will create branches. we should be consistent

Reply:
I took most of it from the existing guidelines we have on Confluence, so I was wondering... The advantage I see it that it may be easier for non-techy persons to make contributions as this is similar process to copying and modifying something, and it avoids us the hassle of guiding them through the branch creation process, checkout, merging.

We can as well keep the fork strategy for external contributors only, and we keep working with branches within the team. But no strong opinion here I'll go with the general consensus :)

Reply:
@senthil10 just realised I still had work to do on this one... :sweat_smile: 
What do you think of Liane's comment though regarding what has been advised in the past?

Reply:
Yes, external people can't create branches anyway I guess (due to repo settings). So yeah, I too agree external collaborators can fork and internal team member can crate new branches 👍

Reply:
Ok then I will let it like it is now, as CONTRIBUTING.md is mostly geared towards external contributors anyway